pkg/vsphere/tasks/waiter.go

```diff
@@ -26,6 +29,10 @@ import (
 	"github.com/vmware/vic/pkg/errors"
 )
 
+const (
+	maxBackoffFactor = int64(8)
```

Comment:
alright, I  have increased it to 16. which has a max of 2400 but that is only after several failures.